NRediSearch/Schema.cs

```diff
@@ -95,7 +114,18 @@ public Schema AddGeoField(string name)
         /// <returns>the schema object</returns>
         public Schema AddNumericField(string name)
         {
-            Fields.Add(new Field(name, FieldType.Numeric));
+            Fields.Add(new Field(name, FieldType.Numeric, false));
+            return this;
+        }
+
+        /// <summary>
+        /// Add a numeric field that can be sorted on
+        /// </summary>
+        /// <param name="name">the field's name</param>
+        /// <returns>the schema object</returns>
+        public Schema AddSortableNumericField(string name)
```

Comment:
Little confusion over here. Are you recommending to change `AddNumericField(string name)` to  `AddNumericField(string name, bool sortable = false)` or no change required as of now?